corehq/apps/export/views.py

```diff
@@ -2053,6 +2041,20 @@ def get(self, request, *args, **kwargs):
             messages.error(request, message)
         return super(BaseEditNewCustomExportView, self).get(request, *args, **kwargs)
 
+    @method_decorator(login_and_domain_required)
+    def post(self, request, *args, **kwargs):
+        try:
+            new_export_instance = self.new_export_instance
+        except ResourceNotFound:
+            new_export_instance = None
+        if (
```

Comment:
@millerdev Server-side validation of user-permissions is applied when editing exports.  👍